src/components/Article.jsx

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState} from "react"
+import axios from "axios"
+import { useParams } from "react-router-dom"
+
+function Article (){
+    const article = useParams()
+    const articleChoice = article.article_id
+    
+    const [oneArticle, setOneArticle] = useState([])
+     useEffect(()=>{
+         axios.get(`https://northcoders-news-api-bjpy.onrender.com/api/articles/${articleChoice}`)
```

Comment:
as you go on you might find extraacting the api calls into a different file and giving the functions good names might benefit you organisationally